---
layout: tool 
title: Cinteny
developer_github_account: 
paper: https://www.ncbi.nlm.nih.gov/pubmed/17343765
paper_title: "Cinteny: flexible analysis and visualization of synteny and genome rearrangements in multiple organisms."
doi: 10.1186/1471-2105-8-82
browser: http://cinteny.cchmc.org
abstract: "BACKGROUND: Identifying syntenic regions, i.e., blocks of genes or other markers with evolutionary conserved order, and quantifying evolutionary relatedness between genomes in terms of chromosomal rearrangements is one of the central goals in comparative genomics. However, the analysis of synteny and the resulting assessment of genome rearrangements are sensitive to the choice of a number of arbitrary parameters that affect the detection of synteny blocks. In particular, the choice of a set of markers and the effect of different aggregation strategies, which enable coarse graining of synteny blocks and exclusion of micro-rearrangements, need to be assessed. Therefore, existing tools and resources that facilitate identification, visualization and analysis of synteny need to be further improved to provide a flexible platform for such analysis, especially in the context of multiple genomes. RESULTS: We present a new tool, Cinteny, for fast identification and analysis of synteny with different sets of markers and various levels of coarse graining of syntenic blocks. Using Hannenhalli-Pevzner approach and its extensions, Cinteny also enables interactive determination of evolutionary relationships between genomes in terms of the number of rearrangements (the reversal distance). In particular, Cinteny provides: i) integration of synteny browsing with assessment of evolutionary distances for multiple genomes; ii) flexibility to adjust the parameters and re-compute the results on-the-fly; iii) ability to work with user provided data, such as orthologous genes, sequence tags or other conserved markers. In addition, Cinteny provides many annotated mammalian, invertebrate and fungal genomes that are pre-loaded and available for analysis at http://cinteny.cchmc.org. CONCLUSION: Cinteny allows one to automatically compare multiple genomes and perform sensitivity analysis for synteny block detection and for the subsequent computation of reversal distances. Cinteny can also be used to interactively browse syntenic blocks conserved in multiple genomes, to facilitate genome annotation and validation of assemblies for newly sequenced genomes, and to construct and assess phylogenomic trees."
citation: "Sinha AU, Meller J. Cinteny: flexible analysis and visualization of synteny and genome rearrangements in multiple organisms. BMC Bioinformatics. 2007;8: 82."
citation_count: 134
feature_type: ['segment sparse']
interconnection_type: ['between']
genome_layout: ['linear']
partition: ['segregated']
abstraction: ['no']
arrangement: linear parallel
view: single
scale: single
focus: single
coordinate_systems: multiple
access_format: [web application]
user_documentation_availability: yes
user_documentation: https://cinteny.cchmc.org/doc/demo.php
pub_year: 2007
license: unavailable
license_form: unknown
supported_files: ['other']
image: /assets/Cinteny.png
image_location: https://cinteny.cchmc.org/doc/wholegenome.php
notes: 
---
