---
layout: tool 
title: Edgar Genome Browser
developer_github_account: 
paper: https://www.ncbi.nlm.nih.gov/pubmed/27098043
paper_title: "EDGAR 2.0: an enhanced software platform for comparative gene content analyses."
doi: 10.1093/nar/gkw255
browser: https://edgar.computational.bio.uni-giessen.de/cgi-bin/edgar.cgi?action=view&type=genomeBrowser&project=EDGAR_Aerococcus
abstract: "The rapidly increasing availability of microbial genome sequences has led to a growing demand for bioinformatics software tools that support the functional analysis based on the comparison of closely related genomes. By utilizing comparative approaches on gene level it is possible to gain insights into the core genes which represent the set of shared features for a set of organisms under study. Vice versa singleton genes can be identified to elucidate the specific properties of an individual genome. Since initial publication, the EDGAR platform has become one of the most established software tools in the field of comparative genomics. Over the last years, the software has been continuously improved and a large number of new analysis features have been added. For the new version, EDGAR 2.0, the gene orthology estimation approach was newly designed and completely re-implemented. Among other new features, EDGAR 2.0 provides extended phylogenetic analysis features like AAI (Average Amino Acid Identity) and ANI (Average Nucleotide Identity) matrices, genome set size statistics and modernized visualizations like interactive synteny plots or Venn diagrams. Thereby, the software supports a quick and user-friendly survey of evolutionary relationships between microbial genomes and simplifies the process of obtaining new biological insights into their differential gene content. All features are offered to the scientific community via a web-based and therefore platform-independent user interface, which allows easy browsing of precomputed datasets. The web server is accessible at http://edgar.computational.bio."
citation: "Blom J, Kreis J, Spänig S, Juhre T, Bertelli C, Ernst C, et al. EDGAR 2.0: an enhanced software platform for comparative gene content analyses. Nucleic Acids Res. academic.oup.com; 2016;44: W22–8."
citation_count: 125
feature_type: ['segment sparse']
interconnection_type: ['no']
genome_layout: ['linear', 'circular']
partition: ['segregated', 'contiguous']
abstraction: ['no']
arrangement: ['linear parallel', 'circular parallel']
view: multiple
scale: single
focus: single
coordinate_systems: multiple
access_format: [web application]
user_documentation_availability: yes
user_documentation: https://www.uni-giessen.de/fbz/fb08/Inst/bioinformatik/software/EDGAR/documentation
pub_year: 2016
license: unavailable
license_form: unknown
supported_files: ['other']
image: /assets/edgargenomebrowser.png
image_location: https://edgar.computational.bio.uni-giessen.de/cgi-bin/edgar.cgi
notes: 
---
