---
layout: tool 
title: HUGIn
developer_github_account: 
paper: https://www.ncbi.nlm.nih.gov/pubmed/28582503
paper_title: "HUGIn: Hi-C Unifying Genomic Interrogator."
doi: 10.1093/bioinformatics/btx359
browser: http://yunliweb.its.unc.edu/HUGIn
abstract: "MOTIVATION: High throughput chromatin conformation capture (3C) technologies, such as Hi-C and ChIA-PET, have the potential to elucidate the functional roles of non-coding variants. However, most of published genome-wide unbiased chromatin organization studies have used cultured cell lines, limiting their generalizability. RESULTS: We developed a web browser, HUGIn, to visualize Hi-C data generated from 21 human primary tissues and cell lines. HUGIn enables assessment of chromatin contacts both constitutive across and specific to tissue(s) and/or cell line(s) at any genomic loci, including GWAS SNPs, eQTLs and cis-regulatory elements, facilitating the understanding of both GWAS and eQTL results and functional genomics data. AVAILABILITY AND IMPLEMENTATION: HUGIn is available at http://yunliweb.its.unc.edu/HUGIn."
citation: "Martin JS, Xu Z, Reiner AP, Mohlke KL, Sullivan P, Ren B, et al. HUGIn: Hi-C Unifying Genomic Interrogator. Bioinformatics. academic.oup.com; 2017;33: 3793â€“3795."
citation_count: 31
feature_type: ['segment sparse', 'segment contiguous', 'point sparse', 'point contiguous']
interconnection_type: ['between']
genome_layout: ['linear']
partition: ['segregated']
abstraction: ['no']
arrangement: ['no']
view: single
scale: single
focus: single
coordinate_systems: single
access_format: ['web application']
user_documentation_availability: yes
user_documentation: https://yunliweb.its.unc.edu/HUGIn/tutorial_page.php
pub_year: 2017
license: unavailable
license_form: unknown
supported_files: ['notapplicable']
image: /assets/hugin.png
image_location: https://yunliweb.its.unc.edu/HUGIn/#plots
notes: 
---
