---
layout: tool 
title: Sashimi Plot
developer_github_account: 
paper: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4542614/
paper_title: "Sashimi plots: Quantitative visualization of RNA sequencing read alignments"
doi: 10.1093/bioinformatics/btv034
browser: https://software.broadinstitute.org/software/igv/Sashimi
abstract: "Motivation: Analysis of RNA sequencing (RNA-Seq) data revealed that the vast majority of human genes express multiple mRNA isoforms, produced by alternative pre-mRNA splicing and other mechanisms, and that most alternative isoforms vary in expression between human tissues. As RNA-Seq datasets grow in size, it remains challenging to visualize isoform expression across multiple samples. Results: To help address this problem, we present Sashimi plots, a quantitative visualization of aligned RNA-Seq reads that enables quantitative comparison of exon usage across samples or experimental conditions. Sashimi plots can be made using the Broad Integrated Genome Viewer or with a stand-alone command line program."
citation: "Katz Y, Wang ET, Silterra J, Schwartz S, Wong B, Mesirov JP, et al. Sashimi plots: Quantitative visualization of RNA sequencing read alignments [Internet]. arXiv [q-bio.GN]. 2013. Available: http://arxiv.org/abs/1306.3466"
citation_count: 8
feature_type: ['segment sparse', 'point sparse']
interconnection_type: ['within']
genome_layout: ['linear']
partition: ['contiguous']
abstraction: ['no']
arrangement: ['no']
view: single
scale: single
focus: single
coordinate_systems: single
access_format: [web application]
user_documentation_availability: yes
user_documentation: https://software.broadinstitute.org/software/igv/Sashimi
pub_year: 2015
license: MIT
license_form: open source
supported_files: ['bed', 'fasta', 'sambamcram', 'vcf', 'other']
image: /assets/Sashimi.png
image_location: https://software.broadinstitute.org/software/igv/Sashimi
notes: 
---
